dids/src/test/kotlin/web5/sdk/dids/methods/dht/DhtTest.kt

```diff
@@ -0,0 +1,169 @@
+package web5.sdk.dids.methods.dht
+
+import com.nimbusds.jose.jwk.Curve
+import net.bytebuddy.pool.TypePool.Resolution.Illegal
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
+import org.xbill.DNS.Message
+import web5.sdk.crypto.Ed25519
+import web5.sdk.crypto.InMemoryKeyManager
+import web5.sdk.crypto.Secp256k1
+import java.security.Signature
+import java.security.SignatureException
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+import kotlin.test.assertTrue
+import kotlin.text.hexToByteArray
+
+class DhtTest {
+
+  @Nested
+  inner class Bep44Test {
+
+    // This test vector matches an example from the go implementation
+    // https://github.com/TBD54566975/did-dht-method/blob/main/impl/pkg/dht/dht_test.go#L47
+    @OptIn(ExperimentalStdlibApi::class)
```

Comment:
Why is this needed? And is it possible to do without it?

Reply:
it's needed to use the method `toHexString` which is required by our test vectors